fiftyone/core/models.py

```diff
@@ -78,6 +81,18 @@ def apply_model(
         skip_failures (True): whether to gracefully continue without raising an
             error if predictions cannot be generated for a sample
     """
+    if isinstance(model, fcm.Task):
```

Comment:
I'd prefer to add an `fiftyone.utils.flash.is_flash_model()` method that is used here, to avoid the need to explicitly import `flash` in this file (even though it is only a `lazy_import`).

It's nice to keep all usages of utils in a single place, similar to how we treat `tensorflow`, `torch`, etc.

Reply:
If we check every model that gets passed into `apply_model()` for if it's a flash model, then anytime that `apply_model` gets called you would need to have flash installed even if we lazily import it. Is there any way around it? Maybe checking if it's a FiftyOne model instead?

Reply:
yeah you definitely need to implement such that Flash doesn't have to be installed.

One way to implement that is to use `inspect.getmro()` + `etau.get_class_name()` so that you can use string comparison to determine if `model` is an instance of `fcm.Task`

Reply:
https://stackoverflow.com/questions/1401661